io/plugins/eu.esdihumboldt.hale.io.json/plugin.xml

```diff
@@ -30,6 +30,19 @@
          <contentType
                ref="eu.esdihumboldt.hale.io.json.gzip">
          </contentType>
+         <providerParameter
+               default="2"
```

Comment:
This looks strange - I don't think there is a `default` attribute on that element. Did you use the Eclipse editor to add this?

Reply:
True, I initially edited the xml  but then later modified it from the editor. Looks like this left the initial changes unaltered. Thank you Simon.